cli/command/trust/sign.go

```diff
@@ -18,19 +18,26 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type signOptions struct {
+	local bool
+}
+
 func newSignCommand(dockerCli command.Cli) *cobra.Command {
+	options := signOptions{}
 	cmd := &cobra.Command{
 		Use:   "sign IMAGE:TAG",
 		Short: "Sign an image",
 		Args:  cli.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runSignImage(dockerCli, args[0])
+			return runSignImage(dockerCli, args[0], options)
 		},
 	}
+	flags := cmd.Flags()
+	flags.BoolVarP(&options.local, "local", "", false, "Sign a locally tagged image")
```

Comment:
I guess this should also use `flags.BoolVar()` now that it doesn't have a shorthand flag.